packages/react/src/components/ContentBlockHorizontal/ContentBlockHorizontal.js

```diff
@@ -17,12 +17,12 @@ const { prefix } = settings;
 /**
  * ContentBlockHorizontal pattern.
  */
-const ContentBlockHorizontal = ({ heading, items }) => {
+const ContentBlockHorizontal = ({ heading, items, border }) => {
```

Comment:
should `border` have a default value to maintain continuity with the currently released version?

Reply:
While setting `border == true` would be the backwards-compatible solution, the functional specs specify that the default value be `false`

https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Layout#4-border

Reply:
got it, thanks for clarifying!